src/main.rs

```diff
@@ -580,31 +579,36 @@ async fn gen_user(jwt: &Jwt, password: &Password) -> Result<UserCreateResult> {
     .await?)
 }
 
-fn gen_jwt(user: &UserCreate) -> Jwt {
+#[derive(Serialize, Deserialize)]
+struct Claims {
+    sub: String,
+    pid: usize,
+    sid: String,
+    kid: usize,
+    iat: u64,
+    exp: u64,
+}
+
+fn gen_jwt(user: &UserCreate) -> Result<Jwt> {
     let start = SystemTime::now();
     let iat_seconds = start
         .duration_since(UNIX_EPOCH)
         .expect("Time before epoch? Something's wrong.")
         .as_secs();
+    let my_claims = Claims {
+        sub: user.user_id.id().to_owned(),
+        pid: user.project_id.0,
+        sid: user.seg_id.0.clone(),
+        kid: user.iak.0,
+        iat: iat_seconds,
+        exp: iat_seconds + 120,
+    };
```

Comment:
Should we maybe export this type out of IronOxide so that it doesn't have to be duplicated everywhere that has a dependency on IronOxide?

Reply:
At first, I totally agreed with this comment, but a normal consumer of ironoxide that isn't creating JWTs on the client wouldn't ever need the `Claims`. One usecase I can think of is someone who is modifying an existing JWT to add the IronCore-specific claims on a backend. In that case, they probably wouldn't want to pull in all of ironoxide (or similar) just to help them add some custom claims.

So in the end I think keeping `Claims` private is probably fine, unless I'm missing some other usecase.

Reply:
Ended up making it public and using it